### DISCUSSION TITLE
Available information

### DISCUSSION THREAD

**Discussion by nashant** [tonyroberts/hawundasmart#26] [General]
Hi, can you tell me what information is actually available in the integration? For instance, modulation level and flow temps?

**Comment by tonyroberts** [comment]
Wundasmart doesn't have that information*, that would come from your boiler. This integration makes your wundasmart rooms available as climate entities and adds temperature and humidity sensors for your wundasmart thermostats.

\* it might be possible via the OpenTherm connection that Wundasmart has, but I don't know (and if it is, it's not currently available through this integration). My boiler's is not OpenTherm compatible so someone else would need to investigate and implement that if it was wanted.
